README.md

```diff
@@ -47,15 +47,19 @@ export type storage = {
     ledger: Contract.NFT.ledger,
     metadata: Contract.TZIP16.metadata,
     token_metadata: Contract.TZIP12.tokenMetadata,
-    operators: Contract.NFT.operators
+    operators: Contract.NFT.operators,
+    extension: unit
 };
 
 type ret = [list<operation>, storage];
+
+import Contract = FA2.NFT
```

Comment:
```suggestion
import Contract = Contract.NFT
```

Reply:
Well I think it's a mistake ? Why are you redefining Contract import ?

Reply:
Thank you, wrong copy paste from the actual example.